Execution failed for task ':app:compileDebugAidl'

**Describe the bug**
Unable to run `yarn android` on Windows machine

**To Reproduce**
Steps to reproduce the behavior:
1. `create-react-native-web-app gitphone`
2. `cd gitphone`
3. `yarn android`
4. See error

```sh
D:\Projects\gitphone>yarn android
yarn run v1.10.1
$ react-native run-android
Scanning folders for symlinks in D:\Projects\gitphone\node_modules (55ms)
JS server already running.
Building and installing the app on the device (cd android && gradlew.bat installDebug)...
Incremental java compilation is an incubating feature.
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:app:prepareComAndroidSupportAppcompatV72301Library UP-TO-DATE
:app:prepareComAndroidSupportSupportV42301Library UP-TO-DATE
:app:prepareComFacebookFbuiTextlayoutbuilderTextlayoutbuilder100Library UP-TO-DATE
:app:prepareComFacebookFrescoDrawee130Library UP-TO-DATE
:app:prepareComFacebookFrescoFbcore130Library UP-TO-DATE
:app:prepareComFacebookFrescoFresco130Library UP-TO-DATE
:app:prepareComFacebookFrescoImagepipeline130Library UP-TO-DATE
:app:prepareComFacebookFrescoImagepipelineBase130Library UP-TO-DATE
:app:prepareComFacebookFrescoImagepipelineOkhttp3130Library UP-TO-DATE
:app:prepareComFacebookReactReactNative0554Library UP-TO-DATE
:app:prepareComFacebookSoloaderSoloader010Library UP-TO-DATE
:app:prepareOrgWebkitAndroidJscR174650Library UP-TO-DATE
:app:prepareDebugDependencies
:app:compileDebugAidl FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugAidl'.
> java.lang.IllegalStateException: aidl is missing

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 12.387 secs
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/getting-started.html
Done in 16.29s.
```

**Expected behavior**
App running without error

--- a/README.md
+++ b/README.md
@@ -156,8 +156,28 @@ $ open ./android/app/build/outputs/apk
 
 [React native docs for IOS](http://facebook.github.io/react-native/releases/0.49/docs/running-on-device.html#building-your-app-for-production)
 
+## Troubleshooting
+If you got `Execution failed for task ':app:compileDebugAidl'` when running `yarn android`, try to update Android Gradle plugin to version 3.1.1 and Gradle to version 4.4.
+
+Here is how to do that:
+1. Open the project in **Android Studio**
+1. Click `Update`
+
+If you got `FAILURE: Build failed with an exception.`, examine the warnings:
+
+1. WARNING: `The specified Android SDK Build Tools version (23.0.1) is ignored, as it is below the minimum supported version (27.0.3) for Android Gradle Plugin 3.1.1.`
+<br />Solution: Update Build Tools
+
+1. WARNING: `Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.`
+<br />Solution: Change `compile` to `implementation`
+
+See issue [#7](https://github.com/VISI-ONE/create-react-native-web-app/issues/7#issuecomment-432263368) for your references.
+
 ## Contributors
-- Or Yoffe https://github.com/orYoffe
+||:octocat:|
+|-|-|
+|Or Yoffe|https://github.com/orYoffe|
+|Havit C. Rovik|https://github.com/haruelrovix|
 
 ## Resources
 - [React Native for Web (react-native-web)](https://github.com/necolas/react-native-web)

--- a/README.md
+++ b/README.md
@@ -160,18 +160,31 @@ $ open ./android/app/build/outputs/apk
 If you got `Execution failed for task ':app:compileDebugAidl'` when running `yarn android`, try to update Android Gradle plugin to version 3.1.1 and Gradle to version 4.4.
 
 Here is how to do that:
-1. Open the project in **Android Studio**
-1. Click `Update`
+1. Open **Android Studio**
+1. Open the `android` project
+<br /><img src="https://cdn-images-1.medium.com/max/800/1*jyLo3Jk-nudieT3aaEzUBQ.png" />
+1. Click `Update` on this prompt
+<br /><img src="https://cdn-images-1.medium.com/max/800/1*7I2tqGZ9C63aUGOtae-XHg.png">
+1. Wait for Android Studio syncing the project.
 
 If you got `FAILURE: Build failed with an exception.`, examine the warnings:
 
 1. WARNING: `The specified Android SDK Build Tools version (23.0.1) is ignored, as it is below the minimum supported version (27.0.3) for Android Gradle Plugin 3.1.1.`
 <br />Solution: Update Build Tools
+<br /><img src="https://cdn-images-1.medium.com/max/1000/1*GUlICoUm4cU4KzUfps3W0Q.png" />
 
 1. WARNING: `Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.`
-<br />Solution: Change `compile` to `implementation`
+<br />Solution: Change `compile` to `implementation` by open `app/build.gradle` file, change `dependencies` section (line 139) to use `implementation` instead of `compile` .
 
-See issue [#7](https://github.com/VISI-ONE/create-react-native-web-app/issues/7#issuecomment-432263368) for your references.
+```java
+dependencies {
+  implementation fileTree(dir: "libs", include: ["*.jar"])
+  implementation "com.android.support:appcompat-v7:23.0.1"
+  implementation "com.facebook.react:react-native:+"
+}
+```
+
+Sync it again and now you can close Android Studio. See issue [#7](https://github.com/VISI-ONE/create-react-native-web-app/issues/7#issuecomment-432263368) for your references.
 
 ## Contributors
 ||:octocat:|

